docs/DeveloperGuide.md

```diff
@@ -451,108 +453,318 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+* 1a. The list is empty.
 
   Use case ends.
 
+
 **Use case: UC08 Create Tags**
 
 **MSS**
 
-1. User creates a tag.
-2. WedLinker displays the successful creation of tag.
+1. User requests to create a tag.
+2. System displays the successful creation of tag.
 
    Use case ends.
 
 **Extensions**
 
-* 2a. The tag already exists.
-  * 2a1. WedLinker does not create a new tag.
-  * 2a2. WedLinker informs the user the tag already exists.
+* 1a. The tag already exists.
+    * 1a1. System does not create a new tag.
+    * 1a2. System informs the user the tag already exists.
+
+      Use case ends.
 
-    Use case ends.
 
 **Use case: UC09 Tagging a contact with a specified tag**
 
 **MSS**
 
-1. User searches for the contact to be tagged (UC06).
-2. WedLinker shows a list of contacts for the search.
-3. User adds the tag to the contact based on the index of the list.
-4. WedLinker informs the user the contact is tagged.
-5. WedLinker shows the user the final result of the contact.
+1. User <ins>lists all contacts (UC01)</ins>.
+2. User adds the tag to the contact.
+3. System informs the user the contact is tagged.
+4. System shows the user the final result of the contact.
 
    Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. The list is empty.
 
   Use case ends.
 
-* 3a. WedLinker detects that the tag does not exist.
+* 2a. System detects that the tag does not exist.
+
+    * 2a1. System <ins>creates a new tag (UC08)</ins>
 
-    * 3a1. WedLinker creates a new tag (UC08)
+      Use case resumes at step 3.
 
-      Use case resumes at step 4
+* 2b. The given index is invalid.
 
-* 3b. The given index is invalid.
+    * 2b1. System shows an error message prompting the user to enter a valid index.
 
-    * 3b1. WedLinker shows an error message prompting the user to enter a valid index.
+      Use case resumes at step 1.
 
 **Use case: UC10 Delete Contact**
 
 **MSS**
 
-1. WedLinker shows a list of contacts containing the name (UC01).
-2. User requests to delete a specific person in the list based on the index.
-3. WedLinker deletes the contact.
+1. User <ins>lists all contacts (UC01)</ins>.
+2. User requests to delete a specific person in the list.
+3. System deletes the contact.
 
    Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. The list is empty.
 
   Use case ends.
 
-* 4a. The given index is invalid.
 
-    * 4a1. WedLinker shows an error message prompting the user to enter a valid index.
+* 2a. The given index is invalid.
 
-      Use case resumes at step 2.
+    * 2a1. System shows an error message prompting the user to enter a valid index.
 
+      Use case resumes at step 1.
+
+
+**Use case: UCSH01 Edit details for a contact**
+
+**MSS**
+
+1. User <ins>lists all contacts (UC01)</ins>
+2. User requests to edit the details of a person and specifies what they want to change the details to
+3. AddressBook changes the existing details to the specified details and shows list of persons with new details
```

Comment:
From the example in the ab3 developer guide they seem to put a period even at the end of each step in the MSS

> Use case: UC06 Search for contacts by Name
> 
> MSS
> 
> 1.    User searches for the contact by name.
> 2.    WedLinker shows a list of contacts containing the name.